### PR TITLE
Added real stations only option

### PR DESCRIPTION
**Pull request by sungeunbae**

### CODE REVIEW

**Review by shot0829 — APPROVED** [review]

File: IM_calculation/scripts/calculate_ims.py [hunk 1]
```diff
@@ -136,6 +136,13 @@ def load_args():
         type=int,
         help="Please provide the number of processors. Default is 1",
     )
+
+    parser.add_argument(
+        "--real_stats_only",
+        action="store_true",
+        help="Please add '-r' to indicate if you want to consider real stations only",
```
Comment (jpa198) [inline IM_calculation/scripts/calculate_ims.py:143]: An explicit '-r' would be nice, but should be covered by argparse automatically adding abbreviations.

**Review by jpa198 — APPROVED** [review]

File: IM_calculation/scripts/calculate_ims.py (hunk 1 above)
Comment (sungeunbae) [reply IM_calculation/scripts/calculate_ims.py:143]: Conflicts with -r --rupture

Comment (jpa198) [reply IM_calculation/scripts/calculate_ims.py:143]: Then the comment is wrong as adding -r is for --rupture, and will cause a crash if someone does try -r for --real_stats_only. Can you change the comment and matching documentation?

Comment (sungeunbae) [reply IM_calculation/scripts/calculate_ims.py:143]: Thanks for catching this!

**Comment by jasonmotha** [comment]
Just for my curiosity, what was the use case for this feature?

**Comment by shot0829** [comment]
> Just for my curiosity, what was the use case for this feature?

Some simulation runs on validation events are ran using non-uniform-grid, instead of just real stations.
The automated workflow or script i created will map stations from observed and bypass this issue, but if you want to manual submit job it will be troublesome .

**Comment by shot0829** [comment]
this change also removes the dependency on having a observed run. so even if you want to just run analysis on a simulated data that has virtual-stations, this change will make it a lot more easier for the user.

**Comment by jpa198** [comment]
> Just for my curiosity, what was the use case for this feature?

The current need is for Vahid to run IMs on large mag validation, which were generated using non uniform grid and so have way more stations than he needs

**Pull request merged**